Blog post about advanced course. (#1205)
root-project/web: blog/_posts/2026-04-20-root-advanced-course.md

---
 title: "ROOT Advanced Course 2026"
 layout: archive
 author: Marta Czurylo
---

We've heard you - at least some of you - and we organized the first edition of the ROOT Advanced Course! 

> The in-person only event took place at CERN in March and was attended by around 40 people. We assumed the participants had the basic familiarity with ROOT and we dived into more advanced topics right away. Similarly to our student course, we also used ROOT from Python via jupyter notebooks. The topics included RNTuples, Unified Histogram Interface (UHI), RooFit, Pythonizations and more advanced features of RDataFrame. We made the course interactive and we were happy to receive many interesting questions from the audience.

Reviewer: ```suggestion
The in-person event took place at CERN in March and was attended by around 40 people. We assumed the participants were already familiar with ROOT and we dived into more advanced topics right away. Similarly to our student course, we used an interactive approach from Python via jupyter notebooks. The topics included RNTuples, Unified Histogram Interface (UHI), RooFit, Pythonizations and more advanced features of RDataFrame. We were happy to receive many interesting questions from the audience.
```
